sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefService.java

```diff
@@ -112,41 +112,50 @@ public List<SclReportItem> updateAllExtRefIedNames(SCL scd) {
     }
 
     @Override
-    public List<SclReportItem> createDataSetAndControlBlocks(SCL scd) {
+    public List<SclReportItem> createDataSetAndControlBlocks(SCL scd, Set<FcdaCsvHelper.FcdaForDataSets> allowedFcdas) {
```

Comment:
this is the only method "createDataSetAndControlBlocks" used.
Maybe we could remove the others ?

Reply:
I think that the other methods are proposed as a library service. Even if we don't use it in RTE side it can help others if needed. Isit ok for you?

Reply:
Yes, this is a requirement from issue #80. The PO wants to execute this service on 3 different scopes : full SCD, single IED or single LDevice. 

I remember arguing with him that we only need to execute it on the full SCD for the moment so no need to implement single IED and single LDevice scopes. He insisted to have all the options. Maybe he has plan to use it in the future.

>  List all ExtRef with their attributes and with their XPath:
>  
>      From SCD file, list all /IED/LDevice/LN0/Inputs/ExtRef
>      If IED.name is specified as input, list all IED/LDevice/LN0/Inputs/ExtRef which reside into the /IED.name
>      If IED.name and LDevice.inst are specified as input, list all IED/LDevice/LN0/Inputs/ExtRef which reside into the /IED.name/LDevice.inst

Reply:
So useless ...
Ok I resolve